stylelint-polaris/plugins/media-queries-allowed-list/index.js

```diff
@@ -73,7 +73,16 @@ const {rule} = stylelint.createPlugin(
           ruleSettings: allowedMediaFeatureNames,
           root,
         },
-        (warning) =>
+        (warning) => {
+          if (
+            // The built-in `media-feature-name-allowed-list` rule
+            // doesn't handle this case:
+            allowedMediaFeatureNames.includes('-ms-high-contrast') &&
+            warning.text.includes('-ms-high-contrast')
+          ) {
+            return;
+          }
```

Comment:
The built-in [`media-feature-name-allowed-list` options](https://stylelint.io/user-guide/rules/list/media-feature-name-allowed-list#options) accepts an `["array", "of", "unprefixed", /media-features/, "regex"]`. However, every combination `-ms-hight-contrast`, `high-contrast`, `/.*high-contrast.*/`, etc. resulted in errors.